### PR TITLE
Blitzshell

### PR DESCRIPTION
**Pull request by MadmanMartian**
A new operator antagonist, functionally just a drone with a gun, sent to the station to complete objectives for whomever has hired it.

Equipment:
 - A Plasma Repeater
 - A Laser Repeater
 - A Tac-knife
 - Remote Uplink for viewing the bounty board
 - An objective-centric gripper for picking up potential bounty objectives
 - A Syringe, for taking blood samples
 - Infinite BSDM, for sending things off for objectives
 - An Agent ID. This is what is used as the access for the drone, so you are encouraged to scan IDs that you find
 - Nanite Container, for self repair. Starts with 3 charges, more can be purchased using uplink points.

Upgrades purchasable through the uplink:
 - [x] More Health
 - [x] Better Cell
 - [x] More Nanite Container charges
 - [x] Speed boost
 - [x] More Blitzshells

### CODE REVIEW

File: code/datums/uplink/uplink_items.dm [hunk 1]
```diff
@@ -68,10 +69,11 @@ var/datum/uplink/uplink = new()
 	// Making the assumption that if no uplink was supplied, then we don't care about antag roles
 	if(!U || !antag_roles.len)
 		return 1
-
 	if(!U.uplink_owner)
+		to_chat(world, "no uplink owner. [antag_roles[1]] [!!length(U.owner_roles & antag_roles)]")
 		return !!length(U.owner_roles & antag_roles)
 	else
+		to_chat(world, "uplink owner, is in antag list? [antag_roles[1]] [player_is_antag_in_list(U.uplink_owner, antag_roles)]")
```
Comment (TorinoFermic) [inline code/datums/uplink/uplink_items.dm:76]: Debug lines ?

Comment (MadmanMartian) [reply code/datums/uplink/uplink_items.dm:76]: yes

Comment (TorinoFermic) [reply code/datums/uplink/uplink_items.dm:76]: Okay, they are going to the world, everyone can hear it if something breaks.

**Comment by MadmanMartian** [comment]
Need to check
 - Drone Control Console
 - Player ability to pick up drones

**Pull request merged**